aip/0142.md

```diff
@@ -69,22 +69,20 @@ message is used. For example, a Python-based tool could convert timestamps to
 [datetime][py_datetime] objects and durations to [timedelta][py_timedelta]
 objects.
 
-### Relative timestamps
+### Relative time segments
 
-In some cases, it may be necessary to represent an absolute point in time
-relative to a different start point than the 1970-01-01 typically adopted by
-UNIX timestamps. In these cases, the [`google.protobuf.Timestamp`][timestamp]
-type **should** still be used, however the field comment **must** explain the
-frame of reference in use.
+In some cases, it may be necessary to represent a time segment inside a stream.
+In these cases, the [`google.protobuf.Duration`][duration] type **should** be
+used, and the field name **should** end with `_offset`.
```

Comment:
It's probably worth reinstating the bit about there being a comment to say what it's relative to.